src/fish.rs

```diff
@@ -11,7 +11,7 @@ use crate::{
 };
 use bevy::prelude::*;
 use rand::{distributions::Standard, prelude::Distribution, Rng};
-use std::slice::Iter;
+use std::{f32::consts::E, slice::Iter};
```

Comment:
I'm not actually sure what `f32::consts::E` is doing here

is there a reason?

Reply:
nope. Random crap I accidentally added